infrastructure/constructs/lambda_config.py

```diff
@@ -11,3 +15,5 @@
     RETENTION_DAYS = aws_logs.RetentionDays.ONE_YEAR
 else:
     RETENTION_DAYS = aws_logs.RetentionDays.THREE_MONTHS
+
+PACKAGING_DIR = tempfile.mkdtemp(dir=BACKEND_DIRECTORY, prefix=".lambda_out_")
```

Comment:
This being at the root of the file means the directory will be created at import time, which isn't ideal. Anything importing this file will create this directory.

Reply:
Have turned this into a function, along with `atexit.register` to remove the directory as per comment above.